### PR TITLE
HARP-10793: Add MapView.mapAnchorsOverlay.

### PR DESCRIPTION
**Pull request by robertoraggi**
Overlayed MapAnchor objects are used to overlay custom three.js
objects on top of the MapView scene. That is, any object added to
`mapView.overlayMapAnchors` is always rendered on top of labels, markers
and any object map object.

### CODE REVIEW

File: @here/harp-mapview/lib/MapView.ts [hunk 1]
```diff
@@ -3840,13 +3879,20 @@ export class MapView extends THREE.EventDispatcher {
...
                     // This is needed so that the target is updated automatically, see:
                     // https://threejs.org/docs/#api/en/lights/DirectionalLight.target
                     this.m_scene.add(directionalLight.target);
                 }
                 this.m_createdLights!.push(light);
+
+                const clonedLight: THREE.Light = light.clone();
+                this.m_overlayScene.add(clonedLight);
+                if (clonedLight instanceof THREE.DirectionalLight) {
+                    this.m_overlayScene.add(clonedLight.target.clone());
```
Comment (nzjony) [inline @here/harp-mapview/lib/MapView.ts:3894]: this isn't needed as far as I can tell, threejs already clones the target. when calling the clone method, see: https://github.com/mrdoob/three.js/blob/92ad8c5f4e8a4aa666343ab92b3fd6260fe9e4ce/src/lights/DirectionalLight.js#L35

File: @here/harp-mapview/lib/MapView.ts [hunk 2]
```diff
@@ -1812,6 +1817,20 @@ export class MapView extends THREE.EventDispatcher {
...
+    /**
+     * The node in this MapView's scene containing the overlayed user [[MapAnchor]]s.
+     *
+     * @remarks
+     * The `MapAnchor`s added to `mapAnchorsOverlay` are rendered on top of the MapView scene.
+     *
+     * All (first level) children of this node will be positioned in world space according to the
+     * [[MapAnchor.geoPosition]].
+     * Deeper level children can be used to position custom objects relative to the anchor node.
+     */
```
Comment (nzjony) [inline @here/harp-mapview/lib/MapView.ts:1829]: would it make sense to add a @beta? here

File: @here/harp-mapview/lib/MapView.ts (hunk 1 above)
Comment (robertoraggi) [reply @here/harp-mapview/lib/MapView.ts]: okay, i removed it.

File: @here/harp-mapview/lib/MapView.ts (hunk 2 above)
Comment (robertoraggi) [reply @here/harp-mapview/lib/MapView.ts:1831]: done.

Comment (ninok) [reply @here/harp-mapview/lib/MapView.ts:1831]: Now we have 3 anchor APIs: `mapAnchors`, `worldAnchors` and `mapAnchorsOverlay`.
Should be merged into one api (e.g. `mapObjects`) with following properties:
- anchor (world or geo space)
- overlay

Nevertheless it's fine for now to have it as a temporary API but we should clean it up asap.

Comment (robertoraggi) [reply @here/harp-mapview/lib/MapView.ts:1864]: i would use just `MapAnchor` or `Anchor`. `MapObject` may not be the correct name, in fact `MapObject` was the initial name we used for this API but then we replaced it with MapAnchor.
https://github.com/heremaps/harp.gl/pull/530

**Pull request closed without merging**